src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -562,6 +563,8 @@ def __init__(
             **sudachi_resource_dir**: (*optional*) string
             **sudachi_dict_type**: (*optional*) string
                 dict type of sudachi, choose from "small", "core", "full".
+            **sudachi_projection**: (*optional*) string
+                Word projection mode of sudachi, choose from "surface", "normalized", "reading", "dictionary", "dictionary_and_surface", "normalized_and_surface", "normalized_nouns".
```

Comment:
```suggestion
            **sudachi_projection**: (*optional*) string
                Word projection mode of sudachi, choose from `["surface", "normalized", "reading", "dictionary", "dictionary_and_surface", "normalized_and_surface", "normalized_nouns"]`.
```

Reply:
Looks nice!
I'd like to change above options for `sudachi_dict_type` in same way.